### PR TITLE
feat: bnb activation payload

### PR DESCRIPTION
**Pull request by brotherlymite**
- [x] Change aave-helpers lib to master after PR merged

### CODE REVIEW

File: src/20231122_AaveV3Bnb_AaveV3BNBActivation/AaveV3Bnb_AaveV3BNBActivation_20231122.sol [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+        stableRateSlope1: _bpsToRay(7_00),
+        stableRateSlope2: _bpsToRay(300_00),
+        baseStableRateOffset: _bpsToRay(2_00),
+        stableRateExcessOffset: _bpsToRay(8_00),
+        optimalStableToTotalDebtRatio: _bpsToRay(20_00)
+      })
+    });
+    listings[2] = IAaveV3ConfigEngine.Listing({
+      asset: BTCB,
+      assetSymbol: 'BTCB',
```
Comment (eboadom) [inline src/20231122_AaveV3Bnb_AaveV3BNBActivation/AaveV3Bnb_AaveV3BNBActivation_20231122.sol:114]: Is this way or `BTCb`?

Comment (brotherlymite) [reply src/20231122_AaveV3Bnb_AaveV3BNBActivation/AaveV3Bnb_AaveV3BNBActivation_20231122.sol:114]: The token symbol on the contract is `BTCB` but maybe `BTCb` is a bit easier to read

File: generator/common.ts [hunk 2]
```diff
@@ -103,7 +103,7 @@ export const CHAIN_TO_CHAIN_ID = {
   Avalanche: avalanche.id,
   Metis: metis.id,
   Base: base.id,
-  Bsc: bsc.id,
+  Bnb: bsc.id,
```
Comment (Zer0dot) [inline generator/common.ts:106]: Should this be bnb.id?

Comment (sendra) [reply generator/common.ts:106]: the .id is given by viem so we can not change

**Review by eboadom — APPROVED** [review]

**Pull request merged**